engine/query-constants/src/main/java/io/deephaven/util/QueryConstants.java

```diff
@@ -241,7 +247,13 @@ private QueryConstants() {}
     /**
      * Minimum finite value of type double.
      */
-    public static final double MIN_FINITE_DOUBLE = Math.nextUp(-Double.MAX_VALUE);
+    /*
+     * Implemented as a constant, so that it can be translated correctly to JS, without needing a proper Math.nextUp()
+     * or nextAfter() implementation that works in JavaScript. Value is generated through running
+     * "Double.toHexString(Math.nextUp(-Double.MAX_VALUE))" in jshell, and tests will validate that this matches the
+     * expected expression.
```

Comment:
```suggestion
     * Implemented as a constant so that it can be translated correctly to JS without needing a proper
     * {@code Math.nextUp()} or {@code Math.nextAfter()} implementation that works in JavaScript. Value is generated by
     * running {@code Double.toHexString(Math.nextUp(-Double.MAX_VALUE))} in jshell, and tests validate that this
     * matches the expected expression.
```